kasa/experimental/sslaestransport.py

```diff
@@ -335,42 +341,36 @@ async def perform_handshake2(self, local_nonce, server_nonce, pwd_hash) -> None:
         _LOGGER.debug("Handshake2 complete ...")
 
     async def perform_handshake1(self) -> tuple[str, str, str]:
-        """Perform the handshake."""
-        _LOGGER.debug("Will perform handshaking...")
+        """Perform the handshake1."""
+        resp_dict = None
+        if self._username:
+            local_nonce = secrets.token_bytes(8).hex().upper()
+            resp_dict = await self.try_send_handshake1(self._username, local_nonce)
+            resp_dict["error_code"] = SmartErrorCode.SESSION_EXPIRED.value
```

Comment:
Feels weird, this is just to force the correct state for handshaking? Worth documenting anyway.

Reply:
Oops, I left this in there for my final device testing. Now removed.